src/Routes.js

```diff
@@ -54,6 +55,7 @@ export const Routes = () => {
             <InsightsRoute path='/' component={ RosPage } rootClass='rosPage' />
             <InsightsRoute path={paths.oops} component={OopsPage} rootClass='oopsPage' />
             <InsightsRoute path={paths.noPermissions} component={NoPermissionsPage} rootClass='noPermissionsPage' />
+            <Route exact path={paths.rosDetailsPage} component={RosDetailsPage} />
```

Comment:
```suggestion
           <InsightsRoute path={paths.rosDetailPage} component={RosDetailsPage} />
```
could you move this line at line number `55` i.e  above line `<InsightsRoute path='/' component={ RosPage } rootClass='rosPage' />
`

Reply:
yeah using `exact` looks like a better option! :+1: